src/ray/object_manager/plasma/store_runner.cc

```diff
@@ -136,6 +122,16 @@ bool PlasmaStoreRunner::IsPlasmaObjectSpillable(const ObjectID &object_id) {
 
 int64_t PlasmaStoreRunner::GetConsumedBytes() { return store_->GetConsumedBytes(); }
 
+int64_t PlasmaStoreRunner::GetFallbackAllocated() const {
+  std::promise<int64_t> p;
+  main_service_.post(
+      [&]() mutable {
+        return p.set_value(allocator_ ? allocator_->FallbackAllocated() : 0);
+      },
+      "PlasmaStoreRunner.GetFallbackAllocatedMemory");
+  return p.get_future().get();
```

Comment:
Isn't this blocking btw?

Reply:
yup it is

Reply:
an alternative is to make those counters std::atomic...

Reply:
This sounds like an better option to me! Hopefully we can remove these issues when solving the threading model problrm…